(scripts) Adds gitter-lockdown

JPRuskin opened the pull request:
When enabled (which it is by default in most choco-bot available rooms), this attempts to remove any new messages. 
This should help our migration to Discord.

I am currently unsure if I should also disable the various Hubot scripts, but as we're leaving it active in a few channels, it seems better to not.

JPRuskin commented:
I don't seem to be able to tag reviewers, but if @gep13 and @designxsteph wouldn't mind looking this over (and correcting my probably terrible use of arrays) that would be amazing?

pauby commented:
I've added those reviewers. Not sure why you can't.

gep13 commented on scripts/gitter-lockdown.coffee:
```diff
@@ -0,0 +1,76 @@
+      '531444a55e986b0712efc533'  # chocolatey/chocolatey
+      '530cd0b45e986b0712efb4cb'  # chocolatey/ChocolateyGUI
+      '54bc1bf1db8155e6700ecc71'  # chocolatey/choco
+      '58b5ba05d73408ce4f4d818c'  # chocolatey/chocolatey-coreteampackages
```
```suggestion
      '58b5ba05d73408ce4f4d818c'  # chocolatey/chocolatey-coreteampackages
      '54e9026715522ed4b3dc4cb6' # chocolatey/puppet-chocolatey
      '58f69dc0d73408ce4f595206' # chocolatey/cChoco
      '5e2f484fd73408ce4fd7ed4b' # chocolatey-community/community
```

JPRuskin replied:
```suggestion
      '58b5ba05d73408ce4f4d818c'  # chocolatey/chocolatey-coreteampackages
      '54e9026715522ed4b3dc4cb6'  # chocolatey/puppet-chocolatey
      '58f69dc0d73408ce4f595206'  # chocolatey/cChoco
      '5e2f484fd73408ce4fd7ed4b'  # chocolatey-community/community
```

JPRuskin replied:
Added choco-bot to those new rooms - good catch, thank you!

JPRuskin replied:
Just a note to please squash the commits when merging this.

JPRuskin committed:
(scripts) Adds gitter-lockdown

When enabled (which it is by default in most choco-bot available rooms), this attempts to remove any new messages. 
This should help our migration to Discord.

gep13 reviewed (approved):
LGTM!

gep13 reviewed (approved):
LGTM!
